buildSrc/src/main/kotlin/PublishConfiguration.kt

```diff
@@ -0,0 +1,35 @@
+@file:Suppress("UnstableApiUsage")
+
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.api.publish.maven.MavenPom
+import org.gradle.kotlin.dsl.assign
+import org.gradle.kotlin.dsl.maven
+
+internal fun PublishingExtension.configureJewelRepositories() {
+    repositories {
+        maven("https://packages.jetbrains.team/maven/p/kpm/public") {
+            name = "Space"
+            credentials {
+                username = System.getenv("MAVEN_SPACE_USERNAME")
+                password = System.getenv("MAVEN_SPACE_PASSWORD")
+            }
+        }
+    }
+}
+
+internal fun MavenPom.configureJewelPom() {
+    name = "Jewel"
+    description = "intelliJ theming system in for Compose."
```

Comment:
Nit: can we fix this? I just noticed we've had a typo for a while :D

```suggestion
    description = "A theme for Compose for Desktop that implements the IntelliJ Platform look and feel."
```